crates/astria-conductor/src/executor.rs

```diff
@@ -114,6 +118,12 @@ struct Executor<C> {
     /// so that we can mark the block as final on the execution layer when
     /// we receive a finalized sequencer block.
     sequencer_hash_to_execution_hash: HashMap<Vec<u8>, Vec<u8>>,
+
+    /// most recently executed sequencer block hash
+    last_safe_block_hash: Hash,
+
+    /// block queue for blocks that have been recieved but their parent has not been executed yet
+    block_queue: PriorityQueue<SequencerBlockData, u64>,
```

Comment:
Can we use the tendermint height instead of the u64 here?

Reply:
Updated.